api-common-java/pom.xml

```diff
@@ -80,6 +80,11 @@
             <version>1.4.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+          <groupId>com.google.j2objc</groupId>
+          <artifactId>j2objc-annotations</artifactId>
+          <version>${j2objc-annotations.version}</version>
```

Comment:
@suztomo We have to declare this dependency here  and the version in `gapic-generator-java-pom-parent`, to resolve a Maven enforcer failure. In addition, it is already declared in third-party-dependencies, do you have a good way to declare this dependency in a centralized place?

Reply:
Use variables. Follow gRPC's version.

Reply:
We are using variables for `errorprone` and `j2objc` in
 - gapic-generator-java-pom-parent
 - java-shared-dependencies/third-party-dependencies